composer.json

```diff
@@ -24,6 +24,7 @@
         "cakephp/cakephp": "^4.2.2"
     },
     "require-dev": {
+        "bedita/core": "^5.0.0",
```

Comment:
Maybe we could promote the `bedita/core` dependency to `require` and remove the check on `FilesystemRegistry`.
Otherwise if we want to keep this plugin free from `bedita/core` dependency I would add `bedita/core` to `suggest` key.

Reply:
I didn't want to add the dependency to avoid headaches with required versions, but it shouldn't be too much of an issue.

The `require-dev` is necessary to have `FilesystemRegistry` in tests. I'd avoid the `suggests` as it's mostly an useless feature, and it doesn't have a built in way to specify the version range. I'd go with the `require` instead if deemed necessary.